### DISCUSSION TITLE
1.0.9.2

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#44] [Обновления]
- Добавлена поддержка m.youtube.com (поддержка старого + нового дизайна, настройки для m.youtube.com не переносятся из настроек youtube.com)
- Фикс отображения кнопки перевода в Twitch Videos (для хромоподобных браузеров)

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.0.9.1...1.0.9.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.0.9.2'>1.0.9.2</a>.</em>

**Comment by ilyhalight** [comment]
В некст обновление постараюсь добавить поддержку twitch player + оставшихся сайтов. Так же, не забывайте писать свои пожелания/идеи о развитие расширения или же проблемы, которые у вас возникли

**Reply by Kerseez** [reply]
Привет! Не нажимается кнопка перевода на Youtube! Переустанавливал скрипт не помогает, раньше всё работало на ура. Сама кнопка есть и три точки есть но не нажимается, когда нажимаешь на неё просто дальше проигрыватель воспроизводит видео.

**Comment by FiLinX** [comment]
Обновил скрипт.
Вобщем да, на десктопной странице твича панель появилась. Но.. Увы там она никак не кликается((
То есть само поле с надписью "Перевести видео" видно что интерактивно - курсор мыши меняет индикацию, но куда б я в неё не тыркал - функция запуска перевода не срабатывает..
Но работает поле где подразумевалось троеточие, по ней разворачивает меню да.. 
А подразумевалось - это потому, что поле есть, а самого троеточия нет) - https://i.ibb.co/fn1bhwK/screenshot-2022-10-26-17-02-48-010.png
На мобильной версии твича панель нормально срабатывает, перевод включается. Троеточие правда тоже отсутствует, но поле функционирует.

Хм, странно, а на ютубе троеточие присутствует)

Упс, а попробовал щас с той десктопной страницы на твиче открыть любой другой видос слева из ленты  -  и панель реагирует уже на них, перевод запускается)  
Странно, я ведь чистил кеш каждый раз.. Видать тот постоянно мной тестируемый линк какой-то замороченный)

**Reply by ilyhalight** [reply]
Про троеточия услышал. Будет исправлено в следующем обновление. Немного недоглядел, что они пропали с твича

**Comment by alexalsp** [comment]
В опере так и не отображается панель в Ютуб. Опять на 1.0.9 перешел.

**Reply by ilyhalight** [reply]
> В опере так и не отображается панель в Ютуб. Опять на 1.0.9 перешел.

Так, а что за версия оперы?

**Reply by alexalsp** [reply]
Версия: 91.0.4516.77

**Reply by alexalsp** [reply]
92.0.4561.33 и sd этой тоже не работает

**Reply by ilyhalight** [reply]
Проверил 91 версию (91.0.4516.20) и 92 версию (92.0.4561.43) оперы - везде работает. Не знаю из-за чего у вас может быть такая проблема :(
![изображение](https://user-images.githubusercontent.com/62353659/200177947-5d22c37d-22d5-4a41-97e9-3c8bfda0111f.png)

**Reply by alexalsp** [reply]
Заработала 1.0.9.2 через [Tampermonkey](https://www.tampermonkey.net/)

Через Violentmonkey не работает хотя 1.0.9.0 через него работает без проблем.

**Reply by ilyhalight** [reply]
> Заработала 1.0.9.2 через [Tampermonkey](https://www.tampermonkey.net/)
> 
> Через Violentmonkey не работает хотя 1.0.9.0 через него работает без проблем.

про виолетманки уже отвечал тут #45

**Reply by alexalsp** [reply]
>  про виолетманки уже отвечал тут

Спасибо. Пока отключил Violentmonkey и переключился на [Tampermonkey](https://www.tampermonkey.net/)

**Comment by sailars** [comment]
Абсолютно случайно наткнулся на данную разработку. БОЛЬШОЕ СПАСИБО за это ПО! Это очень круто. Работает на Firefox 106.0.5 и Vivaldi (Chromium 106).
Вопрос, планируется ли дальше поддерживать переводчик? В смысле не произойдет ли после обновления браузера на след. версию некорректная работа переводчика?

**Reply by ilyhalight** [reply]
>  В смысле не произойдет ли после обновления браузера на след. версию некорректная работа переводчика?

После обновления браузера все будет так же работать. Если обновят сам апи, через который берется перевод, то скорее всего переводчик перестанет работать, в том плане, что будет выдавать "Не удалось получить перевод", и фикс будет зависеть, только, от других энтузиастов (как было в сентября этого года), ибо сам - я сомневаюсь, что смогу такое пофиксить.

**Reply by sailars** [reply]
> > В смысле не произойдет ли после обновления браузера на след. версию некорректная работа переводчика?
> 
> После обновления браузера все будет так же работать. Если обновят сам апи, через который берется перевод, то скорее всего переводчик перестанет работать, в том плане, что будет выдавать "Не удалось получить перевод", и фикс будет зависеть, только, от других энтузиастов (как было в сентября этого года), ибо сам - я сомневаюсь, что смогу такое пофиксить.

Ок, спасибо. Есть несколько пожеланий. Имеет смысл здесь их озвучить?

**Reply by ilyhalight** [reply]
> > > В смысле не произойдет ли после обновления браузера на след. версию некорректная работа переводчика?
> > 
> > 
> > После обновления браузера все будет так же работать. Если обновят сам апи, через который берется перевод, то скорее всего переводчик перестанет работать, в том плане, что будет выдавать "Не удалось получить перевод", и фикс будет зависеть, только, от других энтузиастов (как было в сентября этого года), ибо сам - я сомневаюсь, что смогу такое пофиксить.
> 
> Ок, спасибо. Есть несколько пожеланий. Имеет смысл здесь их озвучить?

Можешь здесь озвучить, почему бы и нет. Если пожелания касаются функционала, то можешь сделать отдельную issue ([тык](https://github.com/ilyhalight/voice-over-translation/issues))

**Comment by FiLinX** [comment]
упс.. сейчас только обнаружил, что звуковая дорожка перевода не останавливается если остановить само видео в плеере)
